package.json

```diff
@@ -17,14 +17,15 @@
   },
   "types": "lib/index.d.ts",
   "scripts": {
+    "prepare": "npm run build",
     "build": "rollup -c",
     "check": "node scripts/check-codestyle.js",
     "release": "node scripts/release.js",
     "prebuild:publish": "npm install && node scripts/prebuild.js --publish",
-    "build:publish": "npm run check && npm run build && npm run test ",
+    "build:publish": "npm run check && npm run test ",
```

Comment:
`build:publish` runs `prebuild:publish` which has `npm install` which runs `prepare` internally and hence `npm run build` will get executed as is.

Reply:
Do we even use this?